internal/ihop/client.go

```diff
@@ -52,27 +52,58 @@ type Image struct {
 
 	Layers []Layer
 
-	unbuffered bool
+	Actual v1.Image
+	Path   string
 }
 
-// ToDaemonImage returns the GGCR v1.Image associated with this Image.
-func (i Image) ToDaemonImage() (v1.Image, error) {
-	ref, err := name.ParseReference(i.Tag)
+func FromImage(img Image) (Image, error) {
```

Comment:
Is it possible to add a unit test for this function?

Reply:
Yes. Thank you for the reminder. This is a place where I had planned to finish a refactoring and then never got to. I've finished that now, removing some fields and methods that are no longer needed.